# üöÄ Who am I?
- My name is Pedro Sebe (he/him)
- I am a data scientist/bioinformatician in Brazil
- I studied [Molecular Sciences](http://www.cecm.usp.br/) at Universidade de S√£o Paulo
- Today I work with Bioinformatics and Data Science at [Varsomics/Hospital Albert Einstein](https://varsomics.com/)
- I am also a grad student at Universidade de S√£o Paulo, studying the mutational signatures of cancer and [how to detect them in clinical samples](https://github.com/PedroSebe/signature-inference)

# ‚ú® My main interests
<details><summary>Bioinformatics</summary>
  Bioinformatics is the interdisciplinary area applying methods from Computer Science to solve Biology problems, specially in Molecular Biology. At Hospital Albert Einstein, I use Bioinformatics tools to analyse data from the clinical lab at Hospital Albert Einstein, in the areas of Oncology (detection of fusions in RNAseq, identification of mutational signatures) and Metagenomics/Metatranscriptomics (identification of pathogens).
</details>
<details><summary>Bayesian models</summary>
  I believe bayesian inference has a lot to offer in the data science/machine learning setting. By using posterior (or posterior predictive distributions), we can represent and assess the uncertainty of individual predictions, which can be quite helpful when models are used in high stakes decisions. Lots of problems from traditional stats/ML can be handled by bayesian inference in a more natural, principled manner, including: 
  <ul>
    <li>partial pooling with hierarchical models</li>
    <li>feature selection with sparsifying priors</li>
    <li>nonparametric curve fitting with gaussian processes</li>
    <li>model selection and model stacking
    <li>missing data imputation by marginalization.
  </ul>
  Despite great advances in computational methods such as Hamiltonian Monte Carlo algorithms, an important problem with bayesian methods is compute time/resources. On the other hand, libraries such as <a href="https://github.com/pymc-devs/pymc">PyMC</a> allows us to implement bayesian models in a modularized and even pythonic way, contributing to the wider adoption of the bayesian framework.
</details>
<details><summary>Machine learning</summary>
  I am also an ML enthusiast. Machine learning is already part of our daily lives, and I believe it will only grow in importance in the next years. It also raises new challenges: how to deploy and maintain our models (MLOps and data engineering), how to ensure data quality, how to explain predictions (explainable AI) and how to ensure our models are fair and not biased. The most interesting ML themes to me right now are feature engineering (how can we translate the relevant information in our data in numerical form for our models in an optimal way?), uncertainty evaluation (how can we build models able to answer "I don't know" for instances where a good prediction cannot be ensured?) and active learning.
</details>
<details><summary>When it all comes together!</summary>
  ML and Bayes both can be applied to generate value in Bioinformatics. Look, for example, at the neural network AlphaFold, that provided the best solution so far to the decades old problem of protein structure prediction. I am glad to be able to work in problems like this at work and grad school! At Varsomics, we are developing ML models to find pathogens in metagenomics/metatranscriptomics data (hopefully will be published soon). In my grad studies, I am trying to recover mutational signatures from smaller sequencing data using bayesian inference.
</details>

# ‚úîÔ∏è My skills
<details><summary>Programming languages</summary>
  <ul>
    <li>Experient in Python and Bash</li>
    <li>Learning R</li>
    <li>Notions of C, JS and PHP</li>
  </ul>
</details>
<details><summary>Python ecosystem for Data Science and Machine Learning</summary>
  <ul>
    <li><b>Data manipulation:</b> Pandas, Numpy</li>
    <li><b>Data visualization:</b> Matplotlib, Seaborn, basic Plotly</li>
    <li><b>Machine Learning:</b> Scikit-learn, Tensorflow, Jax</li>
    <li>Jupyter Notebook/Jupyter Lab, PyMC</li>
  </ul>
</details>
<details><summary>Bioinformatics tools</summary>
  <ul>
    <li>NGS quality control and filtering with AfterQC and FastQC</li>
    <li>De novo assembly with SPAdes</li>
    <li>Variant calling with GATK tools and Freebayes</li>
    <li>Variant annotation with snpEff and Annovar</li>
    <li>Taxonomic classification with Kraken</li>
    <li>Genetic analysis with Plink</li>
    <li>Miscellaneous tasks with samtools and bedtools</li>
  </ul>
</details>
<details><summary>Miscellaneous</summary>
  <ul>
    <li>Version control with <b>Git</b>.</li>
    <li>Basic relational database queries with <b>SQL</b></li>
    <li>Basic queries to graph databases with <b>Neo4j</b></li>
    <li>Learning <b>Tidyverse</b> tools for data analysis in R</li>
  </ul>
</details>

<!--
## üí° What I want to learn next

**PedroSebe/PedroSebe** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
