# Who am I?
My name is Pedro Sebe (he/him), I am a bioinformatician/data scientist in Brazil. I studied Molecular Sciences at Universidade de S√£o Paulo, and today I work with Bioinformatics and Data Science at [Varsomics/Hospital Albert Einstein](https://varsomics.com/). I am also a grad student at Universidade de S√£o Paulo, studying the mutational signatures of cancer and [how to detect them in clinical samples](https://github.com/PedroSebe/signature-inference)

# ‚ú® My main interests
<details><summary>Bioinformatics</summary>
  Bioinformatics is the interdisciplinary area applying methods from Computer Science to solve Biology problems, specially in Molecular Biology. I have experience using Bioinformatics tools to analyse data from the clinical lab at Hospital Albert Einstein, in the areas of Oncology (detection of fusions in RNAseq, identification of mutational signatures) and Metagenomics/Metatranscriptomics (identification of pathogens).
</details>
<details><summary>Bayesian models</summary>
  Bayesian methods allow modelers to combine prior information with empirical data to generate better predictions and inferences. My favorite components that we can use to build bayesian models are Gaussian Process, Hierarchical Models and Sparsifying priors. I believe bayesian inference is also relevant for machine learning, specially when used for high-stakes decisions, where accurate description of uncertainty is crucial. My main tool for exploring Bayes is PyMC.
</details>
<details><summary>Machine learning</summary>
</details>

# ‚úîÔ∏è My skills
<details><summary>Programming languages</summary>
  <ul>
    <li>Experient in Python and Bash</li>
    <li>Learning R</li>
    <li>Notions of C, JS and PHP</li>
  </ul>
</details>
<details><summary>Python ecosystem for Data Science and Machine Learning</summary>
  <ul>
    <li><b>Data manipulation:</b> Pandas, Numpy</li>
    <li><b>Data visualization:</b> Matplotlib, Seaborn, basic Plotly</li>
    <li><b>Machine Learning:</b> Scikit-learn, Tensorflow, Jax</li>
    <li><b>Miscelaneous:</b> Jupyter Notebook/Jupyter Lab, PyMC</li>
  </ul>
</details>
<details><summary>Bioinformatics tools</summary>
  <ul>
    <li>NGS quality control and filtering with AfterQC and FastQC</li>
    <li>De novo assembly with SPAdes</li>
    <li>Variant calling with GATK tools and Freebayes</li>
    <li>Variant annotation with snpEff and Annovar</li>
    <li>Taxonomic classification with Kraken</li>
    <li>Genetic analysis with Plink</li>
    <li>Miscellaneous tasks with samtools and bedtools</li>
  </ul>
</details>
<details><summary>Miscellaneous</summary>
  <ul>
    <li>Version control with <b>Git</b>.</li>
    <li>Basic relational database queries with <b>SQL</b></li>
    <li>Basic queries to graph databases with <b>Neo4j</b></li>
    <li>Learning <b>Tidyverse</b> tools for data analysis in R.</li>
  </ul>
</details>

<!--
## üí° What I want to learn next

**PedroSebe/PedroSebe** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
